hash/spectrum_cass.xml

```diff
@@ -57388,19 +57388,9 @@ license:CC0
 		</part>
 	</software>
 
-	<software name="cybexsys">
-		<description>Cybex (System 4)</description>
-		<year>1987</year>
-		<publisher>System 4</publisher>
-		<part name="cass" interface="spectrum_cass">
-			<dataarea name="cass" size="56332">
-				<rom name="cybex (system 4).tzx" size="56332" crc="2083e44e" sha1="78280081e8db6554507258a40acf59f78b11dc15"/>
-			</dataarea>
-		</part>
-	</software>
 
-	<software name="cybex">
-		<description>Cybex</description>
+	<software name="cybex"> <!-- only works in 48k mode on a 128k machine, not on a 48k machine -->
```

Comment:
This kind of instruction is what `<info name="usage" …` is for.

Reply:
I haven't verified if it's a MAME quirk, if it is then usage doesn't apply here.  For now it's a note.